pkg/operator/ceph/nfs/config.go

```diff
@@ -55,5 +55,19 @@ RADOS_KV {
 	pool = "` + spec.ClientRecovery.Pool + `";
 	namespace = "` + spec.ClientRecovery.Namespace + `";
 }
+
+RADOS_URLS {
+	ceph_conf = '` + cephConfigPath + `';
+	userid = ` + userID + `;
+}
+
+%url	rados://` + spec.ClientRecovery.Pool
+
+	if spec.ClientRecovery.Namespace != "" {
```

Comment:
just a minor readability suggestion... what if you initialized the url before the big const string? then the const string would just use the inline variables instead of using several append operations at the end.

Reply:
Sounds good. Will do.